com.microsoft.mrtk.graphicstools.unity/Samples~/ClippingPrimitives/Prefabs/DescriptionPanel.prefab

```diff
@@ -0,0 +1,469 @@
+%YAML 1.1
```

Comment:
Looks like the material for the backplate is missing: 
![image](https://user-images.githubusercontent.com/13305729/213554572-7d2fe8ab-89f4-4403-8506-63f4d5b85400.png)